atsign-foundation/at_widgets
# refactor: at_mono support

**- What I did**

- Removed `atsign_authentication_helper/` package (discontinued and replaced by `at_onboarding_flutter`.
- Did a mass rename of all other packages from `/` to `/packages/`
- Added `melos.yaml`
- Unncommented ".vscode" in `.gitignore`

Purpose of this PR is to future proof this repo for tools that we need to add using melos, and in at_mono.

**- How I did it**

Local changes

**- How to verify it**

We have nothing path dependent in `.github` or anywhere else in the repo.
Ensure that there are no additions to this PR besides `melos.yaml` and `.gitignore`, everything else should be a delete or rename.

**- Description for the changelog**
refactor: at_mono support
